text_generation/causal_lm/cpp/continuous_batching/library/src/sequence_group.hpp

```diff
@@ -180,6 +180,23 @@ class SequenceGroup {
         m_sequences.erase(remove_it);
     }
 
+    std::vector<int64_t> try_finish_generation() {
+        std::vector<int64_t> dropped_seq_ids;
+        for (auto& running_sequence : get_running_sequences()) {
+            const auto generated_len = running_sequence->get_generated_len();
+            if (m_sampling_params.min_new_tokens > generated_len) {
+                continue;
```

Comment:
since we have EOS logits processor, do we need this check now?
Looks like `running_sequence->get_generated_ids().back() == m_sampling_params.eos_token_id` will not be met until we generate `min_new_tokens`